tests/test_gui.py

```diff
@@ -39,18 +40,17 @@ def test_custom_init(self):
 
     def test_select_auto_points(self):
         self.params.num_points = 3
-        controller = propseg.PropSegController(self.image, self.params)
-        controller.align_image()
-        controller.mode = 'AUTO'
-        expected = [(15, 45, 33, 1), (30, 51, 35, 1), (60, 71, 31, 1)]
-        points = [(x + controller.INTERVAL, y, z) for x, y, z, _ in expected]
+        controller = self._init_auto()
+        expected = [(0, 45, 33, 1), (160, 51, 35, 1), (319, 71, 31, 1)]
```

Comment:
Another reason to have `namedtuple` or something similar here, what is the fourth element on each of these tuples ?